src/tokenizer/definitions.py

```diff
@@ -475,6 +475,9 @@ def make_str(s):
     # Percentage and promille
     "%": ("%", 1.0),
     "‰": ("‰", 0.1),
+    # Velocity
+    "m/s": ("m/s", 1.0),
+    "km/klst.": ("m/s", 0.278),
```

Comment:
How about not including the period in "km/klst"? Including it might be dangerous at the end of a sentence.

Reply:
Done

Reply:
Deleted and added a test for EOS